package.json

```diff
@@ -36,62 +36,45 @@
     "url": "git+https://github.com/ForgeRock/forgerock-javascript-sdk.git"
   },
   "author": "ForgeRock",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/ForgeRock/forgerock-javascript-sdk/issues"
   },
   "homepage": "https://github.com/ForgeRock/forgerock-javascript-sdk#readme",
   "devDependencies": {
-    "@nrwl/cli": "^13.3.9",
-    "@nrwl/eslint-plugin-nx": "^13.3.9",
-    "@nrwl/express": "^13.3.9",
-    "@nrwl/jest": "^13.3.9",
-    "@nrwl/js": "^13.3.9",
-    "@nrwl/linter": "^13.3.9",
-    "@nrwl/node": "^13.3.9",
-    "@nrwl/nx-cloud": "^13.0.0",
-    "@nrwl/react": "^13.3.9",
+    "@nrwl/cli": "13.8.8",
+    "@nrwl/eslint-plugin-nx": "13.8.8",
+    "@nrwl/express": "13.8.8",
+    "@nrwl/jest": "13.8.8",
+    "@nrwl/js": "13.8.8",
+    "@nrwl/linter": "13.8.8",
+    "@nrwl/node": "13.8.8",
+    "@nrwl/nx-cloud": "13.1.6",
+    "@nrwl/react": "13.8.8",
     "@nrwl/schematics": "^8.12.11",
-    "@nrwl/tao": "^13.3.9",
-    "@nrwl/web": "^13.3.9",
-    "@nrwl/workspace": "^13.3.9",
-    "@playwright/test": "^1.17.1",
+    "@nrwl/tao": "13.8.8",
+    "@nrwl/web": "13.8.8",
+    "@nrwl/workspace": "13.8.8",
```

Comment:
A lot of this looks like the same changes from this PR: https://github.com/ForgeRock/forgerock-javascript-sdk/pull/188/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L111-L113. Is there a shared commit or something?

Reply:
Correct. I didn't want to deal with the rebase. I wrote in that PR that this is required. This is the PR that aligns with sample app dependency clean up just in our develop